src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
+  practices: string[];
+}
```

Comment:
PullProofAI

> Note: This review was outside of the patch, but no patch was found that overlapped with it. Original lines [22-22]

The addition of `@IsArray()` decorator is unnecessary as `practices` is already defined as an array of strings. It can be removed.
```diff
-   @IsArray()
```


<!-- This is an auto-generated comment by PullProofAI -->